install_service_systemd.sh

```diff
@@ -76,6 +76,14 @@ WorkingDirectory=${JACKETT_DIR}
 ExecStart=/bin/sh "${JACKETT_DIR}/jackett_launcher.sh"
 TimeoutStopSec=30
 
+ProtectHome=yes
+ProtectSystem=full
+PrivateTmp=yes
+NoNewPrivileges=yes
+ProtectDevices=yes
+ProtectKernelModules=yes
+ProtectKernelTunables=yes
+
```

Comment:
https://www.freedesktop.org/software/systemd/man/systemd.exec.html#ProtectControlGroups=
```suggestion
ProtectControlGroups=yes

```